### PR TITLE
Translating portion of accessing_files.md

### PR DESCRIPTION
**Pull request by singlemocha**
Signed-off-by: Minjung Lee <singlemocha@hotmail.com>

### CODE REVIEW

File: content/ko/docs/chart_template_guide/accessing_files.md [hunk 1]
```diff
@@ -4,26 +4,21 @@ description: "템플릿 안에 있는 파일에 접근하는 방법"
...
-  security reasons.
-  - Files in `templates/` cannot be accessed.
-  - Files excluded using `.helmignore` cannot be accessed.
-- Charts do not preserve UNIX mode information, so file-level permissions will
-  have no impact on the availability of a file when it comes to the `.Files`
-  object.
+이전 항목에서 지명 템플릿을 작성하고 액세스하는 몇 가지 방법을 살펴보았는데 이런
+방법으로 다른 템플릿에서 필요한 템플릿을 쉽게 가져올 수 있다. 하지만 어떤 경우에는
+_템플릿이 아닌 파일_ 을 가져와서 템플릿 렌더러(renderer)를 통해 파일 컨텐츠를 보내지
+않고 직접 삽입해야 할 경우도 있다.
```
Comment (jmnote) [inline content/ko/docs/chart_template_guide/accessing_files.md:10]:
```suggestion
이전 항목에서 지명 템플릿을 만들고 액세스하는 몇 가지 방법을 살펴보았다.
이런 방법으로 다른 템플릿에서 필요한 템플릿을 쉽게 가져올 수 있다.
하지만 때로는 _템플릿이 아닌 파일_ 을 가져와서
그 내용을 템플릿 렌더러(renderer)로 보내지 않고 직접 주입(inject)하려고 하는 경우가
있을 수 있다.
```

File: content/ko/docs/chart_template_guide/accessing_files.md [hunk 2]
```diff
@@ -4,26 +4,21 @@ description: "템플릿 안에 있는 파일에 접근하는 방법"
 weight: 9
```
Comment (jmnote) [inline content/ko/docs/chart_template_guide/accessing_files.md:4]:
```suggestion
weight: 10
```

**Review by jmnote — COMMENTED** [review]
https://raw.githubusercontent.com/helm/helm-www/master/content/en/docs/chart_template_guide/accessing_files.md

수고하셨습니다. 일부 수정해봤습니다.

**Comment by jmnote** [comment]
lgtm

**Comment by jmnote** [comment]
lgtm

**Review by bridgetkromhout — APPROVED** [review]

**Pull request merged**